design-proposals/README.md

```diff
@@ -95,3 +95,4 @@ Next KLIP number: **55**
 | [KLIP-52: BYTES data type support](klip-52-bytes-data-type-support.md)                                                           | Proposal |      0.21.0       |            | [Discussion](https://github.com/confluentinc/ksql/pull/7764) |
 | [KLIP-53: Pull Queries on Streams                                                                                                | Proposal |      0.22.0       |            |                                                              |
 | [KLIP-54: Optimized Range Pull Queries on Tables                                                                                 | Proposal |      0.22.0       |            | [Discussion](https://github.com/confluentinc/ksql/pull/7993) |
+| [KLIP-55: Tablespaces](https://github.com/confluentinc/ksql/blob/af5e34f27627b634a33dc8e5bfa4f64ac75069e6/design-proposals/klip-55-tablespaces.md)                                                                                 | Proposal |      TBD       |            | [Discussion](https://github.com/confluentinc/ksql/pull/8109) |
```

Comment:
I guess the link to the KLIP and the PR link will both be updated when the KLIP itself is populated? Works for me. (Right now the KLIP link is frozen to the current version which is just the placeholder :) )

Reply:
Yep, just threw in something for the moment. Thanks for the callout!